src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./DocumentList.css";
+
+export default function DocumentList({ documents }) {
+  return (
+    <div>
+      <div className="header">
+        <span className="d-none d-md-inline col-md-3">Document Type</span>
+        <span className="d-none d-md-inline col-md-3">File Name</span>
+        <span className="d-none d-md-inline col-md-3">Status</span>
+        <span className="d-none d-md-inline col-md-3">Action (s)</span>
+      </div>
+      <ul>
+        {documents &&
+          documents.map((document) => (
+            <li key={document.name}>
+              <span className="label col-sm-4 d-md-none">Document Type:</span>
+              <span className="col-sm-8 col-md-3">{document.type}</span>
+              <span className="label col-sm-4 d-md-none">File Name:</span>
+              <span className="col-sm-8 col-md-3">{document.name}</span>
+              <span className="label col-sm-4 d-md-none">Status:</span>
+              <span className="col-sm-8 col-md-3">
+                {document.status.description}
+              </span>
+              <span className="label col-sm-4 d-md-none">Action (s):</span>
+              <span className="col-sm-8 col-md-3">withdraw</span>
+            </li>
+          ))}
+      </ul>
+    </div>
+  );
+}
+
+DocumentList.propTypes = {
+  documents: PropTypes.array.isRequired,
```

Comment:
If documents is an array of objects you can get more granular with the PropTypes ie:
```js
documents: PropTypes.arrayOf(PropTypes.object).isRequired
```
or something like:
```js
documents:PropTypes.arrayOf(PropTypes.shape(
   type: PropTypes.string.isRequired,
   ...etc,
)).isRequired
```
Otherwise looks good:)